Services/UI/OJS.Services.Ui.Business/Implementations/SubmissionsBusinessService.cs

```diff
@@ -202,6 +208,8 @@ public async Task<SubmissionDetailsServiceModel> GetDetailsById(int submissionId
         submission.IsEligibleForRetest = await this.submissionsHelper.IsEligibleForRetest(
             submissionId, submission.IsProcessed, submission.IsCompiledSuccessfully, submission.TestRuns.Count);
 
+        submission.AllowMentor = submission.AllowMentor && submission.TestRuns.Any(tr => tr.ResultType != 0);
```

Comment:
Can you use the actual `Enum` instead of `0`

Reply:
Done.